dashboard/src/App.vue

```diff
@@ -90,6 +64,26 @@ export default {
 	font-family: "Avenir", Helvetica, Arial, sans-serif;
 	-webkit-font-smoothing: antialiased;
 	-moz-osx-font-smoothing: grayscale;
+	box-sizing: border-box;
+	display: flex;
```

Comment:
Not sure if this manual management of flexbox stuff is necessary. Have a look at the bulma docs, for example [columns](https://bulma.io/documentation/columns/basics/).

Reply:
Containers are block by default and we don't really want `#app` to be column in its entirety.